libs/documentation/src/lib/components/collapse/demos/basic/collapse-basic.component.html

```diff
@@ -1,11 +1,11 @@
 <div class="text-center">
   <button type="button" class="usa-button text-center" [attr.aria-expanded]="!isCollapsedContent"
-    aria-controls="collapseID" (click)="isCollapsedContent = !isCollapsedContent"> Show / Hide Content
+    aria-controls="collapseID" (click)="isCollapsedContent = !isCollapsedContent" (click)="onClick.emit($event)"> Show / Hide Content
```

Comment:
Do we need to add an event? Playing with it in federalist it looks like each click is causing multiple actions to be registered. The original demo does not alert when the button is clicked, so why are we concerned about it now? If it's clicked it'll toggle, that's your indicator something has happened.